src/gtimelog/timelog.py

```diff
@@ -1048,12 +1048,19 @@ def parse_correction(self, entry):
                     now = None
         if delta_match:
             seconds = int(delta_match.group()) * 60
-            now = datetime.datetime.now().replace(second=0, microsecond=0)
-            now += datetime.timedelta(seconds=seconds)
-            if self.valid_time(now):
-                entry = entry[delta_match.end():]
+            # If positive, offset from the end of the last entry.
+            if seconds >= 0:
+                now = self.window.last_time()
```

Comment:
Now I'm thinking if `-0` and `+0` should be handled differently?  Both seem pretty pointless, so I suppose it doesn't matter.

Reply:
I believe the regular expression won't allow for a zero offset. It just not explicitly shown in the code.